pkg/runtime/cli.go

```diff
@@ -24,32 +24,34 @@ import (
 
 // FromFlags parses command flags and returns DaprRuntime instance
 func FromFlags() (*DaprRuntime, error) {
-	mode := flag.String("mode", string(modes.StandaloneMode), "Runtime mode for Dapr")
-	daprHTTPPort := flag.String("dapr-http-port", fmt.Sprintf("%v", DefaultDaprHTTPPort), "HTTP port for Dapr API to listen on")
-	daprAPIGRPCPort := flag.String("dapr-grpc-port", fmt.Sprintf("%v", DefaultDaprAPIGRPCPort), "gRPC port for the Dapr API to listen on")
-	daprInternalGRPCPort := flag.String("dapr-internal-grpc-port", "", "gRPC port for the Dapr Internal API to listen on")
-	appPort := flag.String("app-port", "", "The port the application is listening on")
-	profilePort := flag.String("profile-port", fmt.Sprintf("%v", DefaultProfilePort), "The port for the profile server")
-	appProtocol := flag.String("protocol", string(HTTPProtocol), "Protocol for the application: grpc or http")
-	componentsPath := flag.String("components-path", "", "Path for components directory. If empty, components will not be loaded. Self-hosted mode only")
-	config := flag.String("config", "", "Path to config file, or name of a configuration object")
-	appID := flag.String("app-id", "", "A unique ID for Dapr. Used for Service Discovery and state")
-	controlPlaneAddress := flag.String("control-plane-address", "", "Address for a Dapr control plane")
-	sentryAddress := flag.String("sentry-address", "", "Address for the Sentry CA service")
-	placementServiceAddress := flag.String("placement-address", "", "Address for the Dapr placement service")
-	allowedOrigins := flag.String("allowed-origins", DefaultAllowedOrigins, "Allowed HTTP origins")
-	enableProfiling := flag.Bool("enable-profiling", false, "Enable profiling")
-	runtimeVersion := flag.Bool("version", false, "Prints the runtime version")
-	maxConcurrency := flag.Int("max-concurrency", -1, "Controls the concurrency level when forwarding requests to user code")
-	enableMTLS := flag.Bool("enable-mtls", false, "Enables automatic mTLS for daprd to daprd communication channels")
+	// Create a new FlagSet and add the flags to that.
+	f1 := flag.NewFlagSet(os.Args[0], flag.ExitOnError)
```

Comment:
can we find the better naming for `f1` ? :)

Reply:
runtimeFlagSet ?